Gaijinizer patch/Map383.txt

```diff
@@ -44,13 +43,12 @@ That's the \\C[2]Marker\\C[0].#DL
 おれさまも、詳しい方法は覚えてないんだ
 > CONTEXT: Map383/events/1/pages/0/14/Dialogue/enemy31
 > CONTEXT: Map386/events/1/pages/0/14/Dialogue/enemy31
-\\C[2]I was told that you should just follow the
-writing on it...\\C[0]
+\\C[2]Apparently you'll be fine as long as you follow
+what's written on them...\\C[0]
 
 
-Fairies don't use those markers to begin
-with, so I can't remember the details of
-how to do it myself.#DL
+But because we fairies don't use those markers to
+begin with, I don't really know how to use them.#Ron
 > END STRING
```

Comment:
We can probably squish this down into one text box now that it doesn't take up too much space.